src/admin-views/widgets/components/radio.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Admin View: Widget Radio Component
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/admin-views/widgets/components/radio.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var string       $label   Label for the radio group.
+ * @var string       $value   Value for the radio.
+ * @var string       $name    Name attribute for the radio.
+ * @var array<array> $options Data for the individual radio buttons.
+ */
+
+if ( ! empty( $label ) ) : ?>
+	<p><?php echo esc_html( $label ); ?></p>
+	<?php
+endif;
```

Comment:
Thinking specifically from a FE and components approach, I feel like this label should be its own thing, it doesn't really have anything to do with the radio.